CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# Note: This is no longer updated and links to omarroths repo, which doesn't exist anymore.

```

Comment:
I though you meant removing email addresses in the changelog, updating repo links is fine.

Reply:
There are too many. Too many that link to the wiki which have to be rewritten to point to the documentation. I don't want to touch that right now.